nuxeo-importer-xml-parser/src/test/java/org/nuxeo/ecm/platform/importer/xml/parser/test/TestMapperServiceMultiFiles.java

```diff
@@ -78,9 +81,21 @@ public void test() throws Exception {
         DocumentModel doc = docs.get(0);
 
         Blob mainFile = (Blob) doc.getPropertyValue("file:content");
+        BufferedReader reader = new BufferedReader(new InputStreamReader(mainFile.getStream()));
+        String result = reader.readLine();
+        assertEquals("file1", result);
```

Comment:
Don't reinvent the wheel, just use `Blob.getString` to get the string content for tests.